src/aol-hb.js

```diff
@@ -27,4 +32,27 @@ globalContext.init = (bidRequestConfig, placementsConfigs) => {
   };
 };
 
+globalContext.queue.push = (cmd) => {
+  if (typeof cmd === 'function') {
+    try {
+      cmd.call();
+    } catch (e) {
+      console.warn('Error processing command', 'aol-hb.js', e);
```

Comment:
1. Theres a possibility that this queue has been initiated as an empty array in the client side in the same global variable page `window.aolhb` before the library has been loaded, so I could initiate as an empty array or as an existent array (that will have functions to execute) from the client side.

2. Yes I override the push method to execute functions, so if client side still uses this queue after the library has been loaded, it straight away executes any function that is pushed.

3. Yes after the library has been loaded and on initiating, I will see if theres any functions to execute on the queue that the user previously added on the client side.

The goal is to allow the user to queue functions before the library has been loaded. In an AMP scenario, in the `remote.html` page, the library is being loaded asynchronously as every resource should be according to AMP standards. 

So on `draw3p`, where magic needs to happen, the library hasn't been loaded yet so we're providing the publishers a easy way to queue code. 
This is also good functionality to have in a normal case, because it allows publishers load the library asynchronously and they can carry on loading the rest of the page.